uiowa/uiowa
# Add P2LB  module and feature split

Building block for an automated move from paragraphs to layout builder. Resolves #1803.

# To test
- `ddev blt ds --site brand.uiowa.edu`
- Observe there are no issues with site sync.
- `ddev drush @brand.local uli`
- Ensure that no P2LB functionality is present on the brand site.

<details>
<summary>Original post</summary>
Process is encapsulated in the sitenow_p2lb module, which adds a config menu at /admin/config/system/sitenow-p2lb. Currently select nodes (ids populated based on presence of paragraphs content on the page), and the option to delete all paragraphs content (tester, or to use following migrate) or update, which runs through the various functions within the main module to move content over. The moved content remains on the same page, so that internal links are preserved.

List of functionality considerations started on #1803.

Couple additional notes:
- Depending on resolution of #1555, some refactoring will be needed. Current build assumes all paragraphs on a site should be removed, which may not be the case.
</details>
